versions.tsp

```diff
@@ -0,0 +1,9 @@
+import "@typespec/versioning";
+
+@Versioning.versioned(OpenAIFlavors)
```

Comment:
I broke out the models into a separate, versioned namespace so the individual `@service`-decorated namespaces could reference the specific version they needed.